helper/src/configpresets/baselines.json

```diff
@@ -126,24 +152,35 @@
                             "cluster": {
                                 "enable_aad": true,
                                 "AksDisableLocalAccounts": true,
-                                "apisecurity": "private",
+                                "apisecurity": "none",
                                 "autoscale": true,
+                                "agentCount": 2,
+                                "maxCount": 5,
                                 "upgradeChannel": "none",
-                                "DefenderForContainers": true
+                                "DefenderForContainers": true,
+                                "AksPaidSkuForSLA": true,
+                                "SystemPoolType": "Standard",
+                                "availabilityZones": "yes"
                             },
                             "addons": {
                                 "networkPolicy": "azure",
                                 "registry": "Premium",
                                 "azurepolicy": "audit",
-                                "ingress": "appgw",
+                                "azurePolicyInitiative": "Restricted",
```

Comment:
I think we might have a few more gaps around policy that we might want to compare at the metadata level.

https://github.com/mspnp/aks-baseline/blob/8ca6ef5b23b5f5fe80bc81933c9336f9ff3c24d7/cluster-stamp.bicep#L72